app/views/teacher_mailer/application_declined.text.erb

```diff
@@ -12,36 +12,34 @@ Thank you for applying for QTS in England. Unfortunately, your application has b
 
 <% end %>
 <% reasons.each do |reason| %>
+## Assessor notes
```

Comment:
@thomasleese this shows as h3 on the front end, how can I make this h4?

Reply:
I think Notify only supports two levels of headings:

https://www.notifications.service.gov.uk/using-notify/formatting#headings-and-subheadings

It doesn't support bold text so I don't think there are any other options here for subheadings.